packages/devtools/src/datamodel.ts

```diff
@@ -326,13 +410,16 @@ export class ModelManager {
     if (this.#ceramic.did == null || !this.#ceramic.did.authenticated) {
       throw new Error('Ceramic instance must be authenticated')
     }
+    if (!isSupportedDID(this.#ceramic.did.id)) {
+      throw new Error('Unsupported DID to create stream for model')
```

Comment:
```suggestion
      throw new Error(`${this.#ceramic.did.id} cannot create a stream for this model.`)
```

Tell the user what DID they're using to help make debugging easier. I can see a world where users use the wrong DID without noticing. This really goes for anywhere that a DID is unsupported.